### PR TITLE
Add back the filter bar to the bc and dc pages

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin/issues/7240
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1309557

### CODE REVIEW

**Comment by jwforres** [comment]
In order to make this look correct I had to drop the gradient from the table header, @ajacobs21e @rhamilto thoughts?

build_config_filter_bar

**Comment by jwforres** [comment]
For reference this is what it looked like with the gradient still there

build_config_filter_gradient

**Comment by jwforres** [comment]
@spadgett can review, will wait to merge until we get the ok on dropping the gradient

**Comment by rhamilto** [comment]
I don't mind the gradient header, but it also looks fine without it.

File: assets/app/styles/_tables.less [hunk 1]
```diff
@@ -91,4 +91,16 @@
       }
     }
   }
+}
+
+.table-filter-wrapper {
+  padding: 10px 10px 5px 10px;
+  border-top: 1px solid #d1d1d1;
+  border-left: 1px solid #d1d1d1;
+  border-right: 1px solid #d1d1d1;
```
Comment (spadgett) [inline assets/app/styles/_tables.less]: Does `@table-border-color` work?

**Comment by spadgett** [comment]
LGTM

File: assets/app/styles/_tables.less (hunk 1 above)
Comment (jwforres) [inline assets/app/styles/_tables.less]: yes, updated

**Comment by ajacobs21e** [comment]
It looks like patternfly uses the gradient. It's a subtle difference but I prefer the gradient.

**Comment by jwforres** [comment]
ok added the gradient back [merge]

**Comment by jwforres** [comment]
rebased [merge]

**Pull request merged**